app/src/main/kotlin/com/hippo/ehviewer/ui/reader/PagerItem.kt

```diff
@@ -95,7 +95,7 @@ fun PagerItem(
                 val grayScale by Settings.grayScale.collectAsState()
                 val invert by Settings.invertedColors.collectAsState()
                 Image(
-                    painter = painter,
+                    painter = remember(painter) { painter },
```

Comment:
Add a comment about DrawablePainter implements RememberObserver and should be remembered it so we won't mess it up in the future.